api/test/trace/trace_state_test.cc

```diff
@@ -7,58 +21,14 @@ namespace
 {
 
 using opentelemetry::trace::TraceState;
+namespace nostd = opentelemetry::nostd;
```

Comment:
This seems not used?

Reply:
Its used in the newly added tests (check line 200). I see that there is some inconsistency, at some places opentelemetry::nostd is used and at some nostd is directly used.

Reply:
I'd suggest keeping its usage consistent.